Makefile

```diff
@@ -0,0 +1,25 @@
+BASE_IMAGE := registry.fedoraproject.org/fedora:29
+TEST_TARGET := ./tests/
+PY_PACKAGE := ogr
+OGR_IMAGE := ogr
+
+build: recipe.yaml
+	sudo ansible-bender build --build-volumes $(CURDIR):/src:Z -- ./recipe.yaml $(BASE_IMAGE) $(OGR_IMAGE)
+
+check:
+	PYTHONPATH=$(CURDIR) pytest-3 -v $(TEST_TARGET)
+
+shell:
+	sudo podman run --rm -ti -v $(CURDIR):/src:Z -w /src $(OGR_IMAGE) bash
+
+check-pypi-packaging:
+	sudo podman run --rm -ti -v $(CURDIR):/src:Z -w /src $(OGR_IMAGE) bash -c '\
```

Comment:
is base_image meant to be used here? won't we run into the same issue you described to me yesterday?

Reply:
just tried this and it works

Reply:
The problem was due to these lines:

```

setuptools.setup(use_scm_version=True,
                 packages=setuptools.find_packages(exclude=['tests*']), )
```

I used `exclude=['tests']`, but that was wrong, beacuse it returns e.g. `tests.unit` and `tests.integration`...

Reply:
This needs fix on the sourcegit side as well.